hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstantTimeGenerator.java

```diff
@@ -97,7 +103,7 @@ public static Date parseDateFromInstantTime(String timestamp) throws ParseExcept
       return Date.from(dt.atZone(ZoneId.systemDefault()).toInstant());
     } catch (DateTimeParseException e) {
       // Special handling for all zero timestamp which is not parsable by DateTimeFormatter
-      if (timestamp.equals(ALL_ZERO_TIMESTAMP)) {
+      if (NOT_PARSABLE_TIMESTAMPS.contains(timestamp)) {
```

Comment:
For each ts, should a different Date be returned, i.e., 
```
INIT_INSTANT_TS = "00000000000000"   => Date(0)
METADATA_BOOTSTRAP_INSTANT_TS = "00000000000001"  => Date(1000)
FULL_BOOTSTRAP_INSTANT_TS = "00000000000002"  => Date(2000)
```
?

Reply:
oh yeah, good idea, I should check if returning a `Date(0)`, `Date(1000)` or `Date(2000)` in method `HoodieActiveTimeline.parseDateFromInstantTime` is safe

let me check the code where using `HoodieActiveTimeline.parseDateFromInstantTime`

Reply:
Hi @yihua 

I checked the code, it is safe to return a value `Date(0)`/`Date(1)`/`Date(2)` because the places we use `HoodieActiveTimeline.parseDateFromInstantTime` and `INIT_INSTANT_TS`/`METADATA_BOOTSTRAP_INSTANT_TS `/`FULL_BOOTSTRAP_INSTANT_TS ` together are for output metrics. So I updated my code, it will return a corresponding value now.

And for clearing this method `HoodieActiveTimeline.parseDateFromInstantTime`, I added more comments on it , and also update the test case in `TestHoodieActiveTimeline.testCreateNewInstantTime`.

Please help to review my code, thanks